test/test_explanation_adapter.py

```diff
@@ -0,0 +1,100 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+import logging
+
+import pytest
+import shap
+from captum.attr import IntegratedGradients
+from common_utils import (create_pytorch_classifier, create_pytorch_regressor,
+                          create_sklearn_random_forest_classifier,
+                          create_sklearn_random_forest_regressor)
+from constants import DatasetConstants, owner_email_tools_and_ux
+from explanation_utils import (
+    validate_global_classification_explanation_shape,
+    validate_global_regression_explanation_shape,
+    validate_local_classification_explanation_shape,
+    validate_local_regression_explanation_shape)
+from interpret_community.adapter import ExplanationAdapter
+from sklearn.model_selection import train_test_split
+from test_serialize_explanation import verify_serialization
+
+try:
+    import torch
+except ImportError:
+    pass
+
+test_logger = logging.getLogger(__name__)
+
+
+@pytest.mark.owner(email=owner_email_tools_and_ux)
+@pytest.mark.usefixtures('clean_dir')
+class TestExplanationAdapter(object):
+
+    def test_working(self):
```

Comment:
Do you still need this test?

Reply:
it seems we have it in a bunch of test files, I just kept a copy of it